packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -98,6 +99,7 @@ module.exports = {
   // In production, we only want to load the polyfills and the app code.
   entry: [require.resolve('./polyfills'), paths.appIndexJs],
   output: {
+    jsonpFunction: camelCase(pkgJson.name + 'JsonpCallback'),
```

Comment:
Is this the same as the default behaviour of Webpack or can this cause regressions?

Reply:
The default value for this function name is `webpackJsonpCallback`. I see a potential opportunity for regressions with funky `package.json` and how lodash's `camelCase` handles them. However, there are some pretty decent rules for npm package names (see https://docs.npmjs.com/files/package.json#name). Plus lodash's camelCase is pretty battle hardened...